frontend/src/concepts/connectionTypes/fields/ConnectionTypeFormFields.tsx

```diff
@@ -48,9 +45,9 @@ const ConnectionTypeFormFields: React.FC<Props> = ({ fields, isPreview, onChange
 
   return (
     <>
-      {fieldGroups?.map((fieldGroup) =>
+      {fieldGroups?.map((fieldGroup, i) =>
         fieldGroup.section ? (
-          <SectionFormField field={fieldGroup.section} key={createKey(fieldGroup.section)}>
+          <SectionFormField field={fieldGroup.section} key={i}>
             {renderDataFields(fieldGroup.fields)}
```

Comment:
Yeah.. we don't have any unique identifiers for the items :(  But here they can also be reordered, so using the index as a key probably shouldn't be done?

Since key is just used by react for tracking the lifecycles I'm pretty sure, I think it wouldn't hurt to use a combination of index and the field properties?

Reply:
Once index is included in the key, there's really no point in concatenating with another string.